email/index.ts

```diff
@@ -0,0 +1,46 @@
+import mjml2html from "mjml";
```

Comment:
curious: Do you need two seperated files for email template - `index.ts` and `index.mjml`?